Dockerfile

```
@@ -135,15 +142,17 @@ RUN  echo " ... adding Openresty, NGINX, and PCRE" \
             --without-http_scgi_module \
             -j${NPROC} \
     && make -j${NPROC} \
-    && make install \
+    && make install
```

If we want to complete this docker layer here, I'd clean it up too:
```suggestion
     && apk del g++ gcc make \
     && rm -rf /var/cache/apk/* \
     && rm -rf /tmp/api-gateway
```

---

pushed the change